src/panel/poi_panel.js

```diff
@@ -216,6 +216,11 @@ PoiPanel.prototype.emptyClickOnMap = function() {
   }
 };
 
+PoiPanel.prototype.showPhone = function() {
+  document.querySelector(".poi_phone_container_hidden").style.display = "none"
```

Comment:
Why not using the template system?

Reply:
You mean re-render all the template, just to show the number? It seemed overkill.
Also, in the category list, the phone number is revealed in the same way (CSS display none => block, and it seemed fine so I did the same)

Reply:
No, just part of the template which is much more maintainable than having to handle the DOM ourselves.